tests/unittest/_torch/modeling/test_nemotron_nano_preprocessing.py

```
@@ -999,3 +999,526 @@ def test_prediction_matches_actual(self, orig_sr, target_sr, audio_length):
+        `{"video": {"evs_ids": tensor}}` in the second slot of the tuple."""
+        proc = self._make_evs_video_processor()
+        vid_ctx = proc.video_context_token_id
+        num_frames = 2
+        prompt = [1, vid_ctx, 2]
+        # Override mock for 2-frame video.
+        proc._compute_token_numbers_per_video = mock.Mock(return_value=[[50, 0]])
+        frames = [Image.new("RGB", (512, 512)) for _ in range(num_frames)]
+        mm_data = {"video": [SimpleNamespace(frames=frames, metadata=None)]}
+        total_mm = 50 + 2 * num_frames
+
+        expanded, mm_data_updates = proc.expand_prompt_token_ids_for_mm(
```

Ruff (0.15.10)

[warning] Unpacked variable `expanded` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)